source/interprocedural_analyses/taint/test/integration/model_query.py.pysa

```diff
@@ -124,6 +124,18 @@ ModelQuery(
   ]
 )
 
+ModelQuery(
```

Comment:
Hmm, you'll need to run the integration tests to get the generated changes after updating the integration test here. Your changes to model_query.py should be included, in addition to the generated changes to model_query.py.models, model_query.py.cg, model_query.py.overrides, in this PR

Reply:
Hey @onionymous 
> I haven't run the integration tests yet, because I don't know how to run them (Running python3 ./scripts/run_integration_tests.sh ./ for instance says that ~/pyre-check/.DS_Store is not directory!